cypress/integration/SourceTest.ts

```diff
@@ -0,0 +1,214 @@
+describe("Source Test", () => {
+  /**
+   * This test suite verifies that sources are possible to modify and set the particular parameters.
+   */
+  describe("Create a Source Processor", () => {
+    const instanceUrl: string =
+      "/instance/830c8f0d-c677-492f-8d7e-0f81893fbba6";
+
+    beforeEach(() => {
+      const processorName: string = "Source processor";
+      cy.visit(`${instanceUrl}/create-processor`);
+      cy.ouiaId("loading-table", "PF4/Card", { timeout: 30000 }).should(
+        "not.exist"
+      );
+      cy.ouiaId("source", "Tile").should("be.visible").click();
+      cy.ouiaId("processor-name", "PF4/TextInput")
+        .should("be.visible")
+        .type(processorName);
+    });
+
+    it("The source's selector contains values", () => {
+      const source = [
+        "Select a source",
+        "Aws S3 Source",
+        "Dummy Source",
+        "Aws Sqs Source",
+        "Slack Source",
+        "Google PubSub Source",
+      ];
+
+      cy.ouiaId("source-type", "PF4/FormSelect")
+        .find("option")
+        .should("have.length", source.length)
+        .each((item, index) => {
+          cy.wrap(item).should("have.text", source[index]);
+        });
+    });
+
+    it("The default state of the configuration", () => {
+      cy.ouiaId("missing-sources", "PF4/TextInput")
+        .scrollIntoView()
+        .should("be.visible")
+        .should("be.disabled");
+      cy.ouiaId("configuration").should("not.exist");
+    });
+
+    describe("Send to Slack", () => {
```

Comment:
minor, I think we should name `Slack Source`

Reply:
@jomarko fixed